draft-wang-ppm-differential-privacy.md

```diff
@@ -549,36 +573,41 @@ class DpMechanism:
 
 > TODO: Specify a Gaussian sampler from Algorithm 3 of {{CKS20}} (#10).
 
-## Binary Randomized Response {#rr}
+## Symmetric RAPPOR {#symmetric-rappor}
+
+This section describes symmetric RAPPOR that was first proposed in {{EPK14}},
```

Comment:
From Kunal: we should at least cite the original proposal of RAPPOR, but [MJTB+22](https://arxiv.org/abs/2211.10082) has a clean description of the symmetric RAPPOR that we use here.